### DISCUSSION TITLE
MercadoPago.js - Form could not be submitted

### DISCUSSION THREAD

**Discussion by SeriusDavid** [mercadopago/sdk-js#173] [Q&A] [answered]
Hi! im presenting this error without any code change on my checkout API

Im looking up for error code but i cant find any that describe this problem

we have running this implementation of the api like 3 years or so
https://www.mercadopago.cl/developers/es/docs/checkout-api/integration-configuration/card/integrate-via-cardform

![image](https://github.com/mercadopago/sdk-js/assets/15957376/3575e5fd-384d-4c0b-b6e1-34427c1e4bbc)
![image](https://github.com/mercadopago/sdk-js/assets/15957376/da0c8441-d8d2-4c1a-ae26-7648f6624be8)

**Comment by lucmkz** [comment]
Good afternoon! Do you have more error details to share? 
I need to be able to simulate this scenario to give you a better support, you are using the cardform, correct? 
this cardform is with secure fields (iframes), correct?

Can you share the application link so I can see the error as it happens and simulate? 
Or maybe share with me the code snippet of your implementation? 

Thanks

**Comment by SeriusDavid** [comment]
Hi, of course!

when you try to make the payment its when the error displays

it displays token handling error to in the javascript console

here's the link: https://www.mudango.com/payment_methods/mercado_pago/client_payment/rYkBrbDbm599tcKiMoDrtUjv

![image](https://github.com/mercadopago/sdk-js/assets/15957376/f95abeb9-5e7c-4cdc-b69b-f41e7348fb46)
![image](https://github.com/mercadopago/sdk-js/assets/15957376/cbd4ac4b-a68f-4e49-988b-ef10b6aed8e7)

**Comment by lucmkz** [comment, marked as answer]
checking your code, I realized that you have more than one instance of the `cardform`, today it is not possible to have more than one instance of `cardform` on the same screen, one of the implementations is displaying for the user, and the other has `display: none`, follow print.

![image](https://github.com/mercadopago/sdk-js/assets/31546923/9bbc7590-5653-45f4-92ed-71f43d9698ff)

The problem is happening, because when making the payment, the validations are being sent from the first `cardform`, (which has `display:none`).

To resolve, it is necessary to remove one of the `cardform` from the screen.

**Reply by SeriusDavid** [reply]
We have been working like this for a year. Is this change new?

**Comment by lucmkz** [comment]
We are always working on improvements to our SDK, so there were probably some changes, yes, but we cannot guarantee the operation of an integration that does not comply with our documentation, thank you for your understanding.

Thanks for getting in touch, see you later!

**Reply by SeriusDavid** [reply]
thanks for the quick reply and for the support!
